packages/fuel-indexer-benchmarks/benches/wasm/examples/fuel_explorer.rs

```diff
@@ -0,0 +1,144 @@
+use criterion::Criterion;
+use fuel_indexer_benchmarks::{create_wasm_indexer_benchmark, create_wasm_manifest};
+use fuel_indexer_lib::{config::IndexerConfig, manifest::Manifest};
+
+fn setup_fuel_explorer_manifest() -> Manifest {
+    create_wasm_manifest(
+        "indexer_benchmarks",
+        "fuel_explorer",
+        "examples/fuel-explorer/fuel-explorer/schema/fuel_explorer.schema.graphql",
+        "target/wasm32-unknown-unknown/release/fuel_explorer.wasm",
+    )
+}
+
+// The `criterion_group!` macro requires that target functions have a name.
+// Thus, each target function is actually a wrapper around a closure in
+// order to prevent code duplication.
+
+pub(crate) fn fuel_explorer_10_blocks_start_1(c: &mut Criterion) {
+    let f = create_wasm_indexer_benchmark(1, 10, "fuel_explorer_10_blocks_start_1");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_100_blocks_start_1(c: &mut Criterion) {
+    let f = create_wasm_indexer_benchmark(1, 100, "fuel_explorer_100_blocks_start_1");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_1000_blocks_start_1(c: &mut Criterion) {
+    let f = create_wasm_indexer_benchmark(1, 1000, "fuel_explorer_1000_blocks_start_1");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_10_blocks_start_50000(c: &mut Criterion) {
+    let f =
+        create_wasm_indexer_benchmark(50000, 10, "fuel_explorer_10_blocks_start_50000");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_100_blocks_start_50000(c: &mut Criterion) {
+    let f =
+        create_wasm_indexer_benchmark(50000, 100, "fuel_explorer_100_blocks_start_50000");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_1000_blocks_start_50000(c: &mut Criterion) {
+    let f = create_wasm_indexer_benchmark(
+        50000,
+        1000,
+        "fuel_explorer_1000_blocks_start_50000",
+    );
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_10_blocks_start_200000(c: &mut Criterion) {
+    let f =
+        create_wasm_indexer_benchmark(200000, 10, "fuel_explorer_10_blocks_start_200000");
+    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
+}
+
+pub(crate) fn fuel_explorer_100_blocks_start_200000(c: &mut Criterion) {
+    let f = create_wasm_indexer_benchmark(
+        200000,
```

Comment:
Mentioned this to Maciej as well, but anyway to remove these magic numbers? 🥲

Reply:
Unfortunately not. 😢 `criterion` requires functions to be named it seems; my first pass at this actually tried to use a closure to generate all of the benchmarking functions from command-line args, but I couldn't get it to work. I'm sure there's _some_ way; if you'd rather me investigate a final time before approving it, I wouldn't mind.

Reply:
Let me clarify. We can't change this:

```rust
pub(crate) fn fuel_explorer_100_blocks_start_50000(c: &mut Criterion) {
    let f =
        create_wasm_indexer_benchmark(50000, 100, "fuel_explorer_100_blocks_start_50000");
    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
}
```

To this:

```rust
pub(crate) fn fuel_explorer_100_blocks_start_50000(c: &mut Criterion) {
    let first_named_thing = 50000;
    let second_named_thing = 100;
    let f =
        create_wasm_indexer_benchmark(first_named_thing, second_named_thing, "fuel_explorer_100_blocks_start_50000");
    f(c, setup_fuel_explorer_manifest(), IndexerConfig::default())
}
```

Just trying to minimize "what do these numbers mean" as much as possible.